packages/plugin-commons/web/src/Base/createBaseMediaPlugin.jsx

```diff
@@ -43,33 +44,35 @@ const handleUploadStart = (
   itemPos
 ) => {
   if (file) {
-    fileReader(file).then(url => {
-      const { type, commonPubsub } = props;
-      const tempData = tempDataBuilder[type]?.({
-        url,
-        file,
-      });
-      onLocalLoad?.(tempData);
-      const handleFileUpload = uploadFunctionGetter[type](props);
-      const {
-        helpers: { onMediaUploadStart, onMediaUploadEnd },
-      } = props;
-      const uploadBIData = onMediaUploadStart(type, file.size, file.type);
-      const fileType = getGalleryFileType(file.type);
-      handleFileUpload(file, ({ data, error }) => {
-        onMediaUploadEnd(uploadBIData, error);
-        error && commonPubsub.set('onMediaUploadError', error);
-        handleUploadFinished(
-          type,
-          getComponentData,
-          data,
-          error,
-          onUploadFinished,
-          itemPos,
-          fileType
-        );
-      });
-    });
+    const { type, commonPubsub } = props;
+    fileReader(file)
+      .then(url => {
+        const tempData = tempDataBuilder[type]?.({
+          url,
+          file,
+        });
+        onLocalLoad?.(tempData);
+        const handleFileUpload = uploadFunctionGetter[type](props);
+        const {
+          helpers: { onMediaUploadStart, onMediaUploadEnd },
+        } = props;
+        const uploadBIData = onMediaUploadStart(type, file.size, file.type);
+        const fileType = getGalleryFileType(file.type);
+        handleFileUpload(file, ({ data, error }) => {
+          onMediaUploadEnd(uploadBIData, error);
+          error && commonPubsub.set('onMediaUploadError', error);
+          handleUploadFinished(
+            type,
+            getComponentData,
+            data,
+            error,
+            onUploadFinished,
+            itemPos,
+            fileType
+          );
+        });
+      })
+      .catch(() => commonPubsub.set('onMediaUploadError', { key: MediaUploadErrorKey.GENERIC }));
```

Comment:
let display the error
```suggestion
      .catch(e => {
          console.error('handleUploadStart error', e);
          commonPubsub.set('onMediaUploadError', { key: MediaUploadErrorKey.GENERIC }));
      }
```

Reply:
done